app/stacks/global/front-door/front-door.tf

```diff
@@ -80,6 +80,16 @@ resource "azurerm_frontdoor" "common" {
     web_application_firewall_policy_link_id = azurerm_frontdoor_firewall_policy.back_office_applications_frontend.id
   }
 
+  dynamic "frontend_endpoint" {
+    for_each = local.back_office_frontend.frontend_endpoint_old == null ? [] : ["apply"]
+
+    content {
+      name                                    = format("%s-%s", local.back_office_frontend.frontend_name, "old")
+      host_name                               = local.back_office_frontend.frontend_endpoint_old
+      web_application_firewall_policy_link_id = azurerm_frontdoor_firewall_policy.back_office_applications_frontend.id
+    }
+  }

```

Comment:
I think the ordering of the frontends matters (see comment on line 68). I think we move this to the end of the list.